src/Ad5001/BetterGen/populator/FallenTreePopulator.php

```diff
@@ -45,29 +46,35 @@ public function populate(ChunkManager $level, $chunkX, $chunkZ, Random $random)
 		$this->level = $level;
 		$amount = $this->getAmount($random);
 		$tree =  TreePopulator::$types[$this->type];
-		$fallenTree = new FallenTree(new $tree());
+		$fallenTree = new \Ad5001\BetterGen\structure\FallenTree(
+			new $tree()
+		);
```

Comment:
1st: a class error. Second: because the whole formatting was messed up anyways. Here it actually was a "debug" to get the correct error line (Cannot use tree as tree because already...), but fixed though